fastify.js

```diff
@@ -247,8 +248,18 @@ function build (options) {
 
   fastify.setErrorHandler = setErrorHandler
 
+  // Set fastify initial configuration options read-only object
+  fastify.initialConfig = getSecuredInitialConfig()
+
   return fastify
 
+  function getSecuredInitialConfig () {
```

Comment:
I think this method should really be in the "initialConfigValidation" file. We can have a signature:

```js
fastify.initialConfig = getSecuredInitialConfig(options)
```

Reply:
done ^^